### ISSUE TITLE
lvm commands don't work on RHEL7 until you start the lvm2-lvmetad service

### ISSUE THREAD

**Issue by irvingpop** [opscode-cookbooks/lvm#43]
When you install the lvm2 package on RHEL7 and try to run a command, you get weird output like:
```
[root@ip-172-31-40-236 etc]# vgs
  /run/lvm/lvmetad.socket: connect failed: No such file or directory
  WARNING: Failed to connect to lvmetad: No such file or directory. Falling back to internal scanning.
  /run/lvm/lvmetad.socket: connect failed: No such file or directory
  /run/lvm/lvmetad.socket: connect failed: No such file or directory
  No volume groups found
```

which causes lvm providers to hang indefinitely.   The fix, on RHEL7 platforms, is to start and enable the `lvm2-lvmetad` service:
```
# chkconfig lvm2-lvmetad on
# service lvm2-lvmetad start
```

**Comment by juliandunn** [comment]
You want to send a PR, @irvingpop?

**Comment by irvingpop** [comment]
The two RHEL7 issues I found with this cookbook are pretty deep in my backlog, but that will change as soon as the next Server release probably :)

I was also hoping for some external confirmation - are others experiencing this with RHEL7 or is it just me?

**Comment by irvingpop** [comment]
this was fixed by #52
